spec/support/vcr.rb

```diff
@@ -10,7 +10,7 @@
   c.ignore_request do |request|
     URI(request.uri).port == 9200
   end
-  c.default_cassette_options = { record: :none, match_requests_on: [:path] }
+  c.default_cassette_options = { record: :once, match_requests_on: [:path] }
```

Comment:
Just curious, why was this change necessary? 🙏

Reply:
@Tuubz It means that if the vcr file is not available, we will need to record (cache) the request and response at least 1 time for the next execution. (Ref: https://github.com/vcr/vcr#usage)

Reply:
> It means that if the vcr file is not available

You mean like removing `vcr: 'services/search/google/valid'`, right?

I was expecting it to record the request after removing it, but it doesn't 

Please let me know if I understood incorrectly 🙇

Reply:
> I was expecting it to record the request after removing this from the test, but it doesn't

It will generate the file, that's why we have to change to `:once` 😄